index.html

```diff
@@ -129,7 +160,7 @@ <h3 class="work-name">Multi-Post Stories</h3>
               <li class="gird">Bootstrap</li>
               <li class="grid0">Ruby</li>
             </ul>
-            <button>See Project</button>
+            <button class="btn-window" id="work1">See Project</button>
           </div>
         </div>
         <div class="work2 flex">
```

Comment:
- [ ] As per project requirement, The projects section is created dynamically with the information from the projects array. Let's try to fix this issue by removing already created HTML code for the projects list and instead it should be added via JavaScript.

![image](https://user-images.githubusercontent.com/53019916/131877710-d43f035c-da95-4dfa-ad2e-3b9bf937b516.png)